acceptance/tests/resource/service/service_enable_linux.rb

```diff
@@ -59,7 +59,11 @@
   apply_manifest_on(agent, manifest_install_package, :catch_failures => true)
 
   step "ensure enabling service creates the start & kill symlinks"
-  is_sysV = ((platform == 'centos' || platform == 'el') && majrelease < 7) ||
+  # amazon linux is based on el-6 but uses the year for its majrelease
+  # version. The condition for a version > 2016 should be removed and
+  # replaced with a discrete amazon condition if/when Beaker understands
+  # amazon as a platform. BKR-1148
+  is_sysV = ((platform == 'centos' || platform == 'el') && (majrelease < 7 || majrelease > 2016)) ||
```

Comment:
puppet is able to manage service resources on Amazon Linux via the default provider as expected.

Example
```
[root@ec2-52-11-130-43 ~]# cat /opt/puppetlabs/puppet/VERSION
1.10.4.36.g1bc805d
[root@ec2-52-11-130-43 ~]# facter os
{
  architecture => "x86_64",
  family => "RedHat",
  hardware => "x86_64",
  name => "Amazon",
  release => {
    full => "2017.03",
    major => "2017",
    minor => "03"
  },
  selinux => {
    enabled => false
  }
}
[root@ec2-52-11-130-43 ~]# puppet resource service puppet
service { 'puppet':
  ensure => 'stopped',
  enable => 'false',
}
[root@ec2-52-11-130-43 ~]# puppet resource service puppet ensure=running
Notice: /Service[puppet]/ensure: ensure changed 'stopped' to 'running'
service { 'puppet':
  ensure => 'running',
}
[root@ec2-52-11-130-43 ~]# puppet resource service puppet
service { 'puppet':
  ensure => 'running',
  enable => 'false',
}
```